ajax/libs/rpg-awesome

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "rpg-awesome",
+  "filename": "rpg-awesome.min.css",
+  "description": "A fantasy themed font and CSS toolkit.",
+  "keywords": [
+    "rpg",
+    "font",
+    "awesome",
+    "css",
+    "toolkit"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/nagoshiashumari/Rpg-Awesome.git"
+  },
+  "authors": [
+    "Daniela Howe",
+    "Ivan Montiel <idmontie@gmail.com>"
+  ],
+  "license": "MIT",
+  "homepage": "https://github.com/nagoshiashumari/Rpg-Awesome",
```

Comment:
Let's use this link https://nagoshiashumari.github.io/Rpg-Awesome/ as the homepage.
Thanks.